src/test/runtest.sh

```diff
@@ -117,10 +117,12 @@ function MpirunString
          ;;
       ray*)
          shift
-         RunString="lrun -n$*"
+         RunString="mpirun -n $*"
```

Comment:
Why mpirun here and lrun on lassen?

Reply:
I would like to use `lrun` on ray too, but it has some weird MPI errors (which has been reported to LC). Maybe will change it back when it is fixed.

Reply:
I see.  This makes sense.  Thanks again!